Tasks/Common/azurerest-common/azurerestutility.ts

```diff
@@ -211,8 +211,8 @@ export async function getAzureRMWebAppConfigDetails(SPN, webAppName: string, res
             deferred.resolve(obj);
         }
         else {
-            tl.error(response.statusMessage);
-            deferred.reject(tl.loc('UnabletoretrieveAzureRMWebAppConfigDetails', response.statusCode, response.statusMessage));
+            tl.error(body);
```

Comment:
'body' may have some context regarding the error. it will be be a string or just a simple json.